createdisk.sh

```diff
@@ -53,9 +53,6 @@ VM_IP=$(sudo virsh domifaddr ${VM_NAME} | tail -2 | head -1 | awk '{print $4}' |
 wait_for_ssh ${VM_NAME} ${VM_IP}
 
 if [ ${BUNDLE_TYPE} != "microshift" ]; then
-    # Remove unused images from container storage
-    ${SSH} core@${VM_IP} -- 'sudo crictl rmi --prune'
-    
     # Disable kubelet service
     ${SSH} core@${VM_IP} -- sudo systemctl disable kubelet
     
```

Comment:
>isn't the kubelet service strongly tied to the kubeapi server?

@cfergeau if you mean when stopping this service should also stop the apiserver then no,  because pods are still running.